theano/sandbox/cuda/opt.py

```diff
@@ -2659,15 +2643,15 @@ def gpu_sparse_block_outer_opt(node):
 
         inputs = _clear_host_from_gpu(node.inputs)
 
-        return [host_from_gpu(GpuSparseBlockOuter(node.op.inplace)(*inputs))]
+        return [host_from_gpu(GpuSparseBlockOuter()(*inputs))]
```

Comment:
@nouiz why this change ? It was only Dimshuffle that was made not to use the inplace right ?